pkg/oc/admin/policy/modify_roles.go

```diff
@@ -564,7 +564,7 @@ func (o *RoleModificationOptions) RemoveRole() error {
 		roleBinding.Subjects, cnt = removeSubjects(roleBinding.Subjects, subjectsToRemove)
 		found += cnt
 
-		if len(roleBinding.Subjects) > 0 {
+		if len(roleBinding.Subjects) > 0 || roleBinding.Annotations[ReconcileProtectAnnotation] == "true" {
```

Comment:
I think we can safely assume that `roleBinding.Annotations` is non-nil?

Reply:
from https://blog.golang.org/go-maps-in-action
```[..]A nil map behaves like an empty map when reading,[..]```

Reply:
Ah I always forget about that.

Reply:
(I get the panic from writing to a nil map mixed up the lack of a panic from a read)